suzieq/poller/controller/utils/inventory_utils.py

```diff
@@ -161,3 +180,17 @@ def get_inventory_config(conf_type: str, inventory: dict) -> dict:
                 configs[name] = conf_obj
 
     return configs
+
+
+def get_env_var(name: str, default: str = '') -> str:
```

Comment:
Better to have a function for parsing the password here instead of one getting the environment variable